spec.go

```diff
@@ -12,4 +12,5 @@ type Interface interface {
 	DeleteBackend(ID string) error
 	GetBackend(ID string) (*vaultapi.MountOutput, error)
 	GetCACertificate(ID string) (string, error)
+	ListBackends() ([]*vaultapi.MountOutput, error)
```

Comment:
The idea is to have a cleanup task in `cert-operator` so we list all Tenant Cluster related PKI backends and check if we have actual Tenant Clusters for them. If not we delete the PKI backend.